ln-visualizer/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_LNVISUALIZER_WEB_IP
+      APP_PORT: 80
+
+  web:
+    image: maxkotlan/ln-visualizer-web:v0.0.20@sha256:ab897a98a5b3640f8e26d423ef505537042a1ebd2804ef540d0ecbc88dc562c4
```

Comment:
The digest here appears to be for a specific arch. If you run `docker pull maxkotlan/ln-visualizer-web:v0.0.20` then at the bottom it will give the digest for multiple architectures. In this case it should be: `sha256:9e4fff7600666ce27d1c835fa66e4a3fb34aabd7018506d749134baf9e8bc481`

Reply:
Ah that makes more sense. I wasn't sure which to use.
Changed to `sha256:9e4fff7600666ce27d1c835fa66e4a3fb34aabd7018506d749134baf9e8bc481`